libraries/core-react/src/components/Typography/Typography.tsx

```diff
@@ -52,7 +52,7 @@ const findTypography = (
 }
 
 const findColor: (a: ColorVariants | string) => string = (
-  inputColor = null,
+  inputColor = 'currentColor',
```

Comment:
❗ Note to self: Major change

Reply:
Reverted this change, and passed `currentColor` via the `color` prop on `<Typography />` instead as this change had unintended consequences.